Tasks/ServiceFabricUpdateAppVersions/Get-VstsBuild.psm1

```diff
@@ -0,0 +1,87 @@
+﻿function Get-VstsBuild
+{
+    [CmdletBinding()]
+    [OutputType([string])]
+    Param
+    (
+        [Parameter(Mandatory=$true)]
+        [ValidateNotNullOrEmpty()]
+        [string]
+        $PkgArtifactName,
+
+        [Parameter(Mandatory=$true)]
+        [ValidateSet("Specific", "LastSuccessful")]
+        [string]
+        $CompareType,
+
+        [string]
+        $BuildNumber
+    )
+
+    $vstsEndpoint = Get-VstsEndpoint -Name SystemVssConnection -Require
```

Comment:
I investigated using that initially, but could never get it to work. I'm sure I was doing something wrong, but I found that the REST API had much better docs and was easier to debug.

If there's a functional difference I can investigate the SDK again, but otherwise I would prefer to leave it as-is.